tests/unit/bokeh/embed/test_standalone.py

```diff
@@ -347,6 +348,46 @@ def test_empty_document(self) -> None:
         with pytest.raises(ValueError, match="Document has no root Models"):
             bes.file_html(doc, CDN)
 
+    def test_resources(self, test_plot: figure) -> None:
+        cdn_url = "https://cdn.bokeh.org/bokeh/"
+        server_url = "http://localhost:5006/static/js/bokeh.js"
```

Comment:
Is hard-coding this here ideal? It could probably be extracted from `bokeh.server`, unless you are intending this as an additional implicit check on the URL value itself.